app/javascript/containers/ReviewModeContainer.jsx

```diff
@@ -17,29 +17,15 @@ import CategoryHeader from "../components/CategoryHeader";
 
 import { post, patch } from "../fetch/requester";
 
-function mapCategorytoQuestions(categoryMap, categoryId, building) {
-  return categoryMap[categoryId].map(question => {
-    // Only display non-dependent questions initially
-    if (question.parent_option_id) {
-      //TODO: look at this later
-      // let filteredQuestion = categoryMap[categoryId].filter((pQuestion) => {
-      //   return (Object.keys(pQuestion.options).map(i => parseInt(i)).includes(question.parent_option_id));
-      // })[0];
-      // let option = building.answers[filteredQuestion.id].selected_option_id;
-      // if (option && option != question.parent_option_id) {
-      //   return null;
-      // }
-      return null;
-    }
-    return (
-      <QuestionContainer
-        mode="review"
-        key={question.id}
-        building_id={building.id}
-        {...question}
-      />
-    );
-  });
+async function postDelegations(delegations) {
```

Comment:
This was deleted in a past PR and logic for delegation requests is in `DelegationRequests.js`, does it need to be added back for something? Otherwise probs delete it